### PR TITLE
Fix lint reporting

### PR DESCRIPTION
**Pull request by therve**
We need to make it fail.

### CODE REVIEW

File: datadog/tests/resource_datadog_integration_slack_channel_test.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func testAccCheckDatadogIntegrationSlackChannelConfig_Create(uniq string) string
    `, uniq)
 }
 
-func testAccCheckDatadogIntegrationSlackChannelConfig_Update(uniq string) string {
+func testAccCheckDatadogIntegrationSlackChannelConfigUpdate(uniq string) string {
```
Comment (nmuesch) [inline datadog/tests/resource_datadog_integration_slack_channel_test.go:86]: Do cassettes need recording if these change?

Comment (therve) [reply datadog/tests/resource_datadog_integration_slack_channel_test.go:86]: No it's private helper.

**Review by zippolyte — APPROVED** [review]

File: datadog/tests/resource_datadog_integration_slack_channel_test.go (hunk 1 above)
Comment (nmuesch) [reply datadog/tests/resource_datadog_integration_slack_channel_test.go:86]: Ahh right, I saw testAcc and assumed 🙂 
LGTM then 👍

**Review by nmuesch — APPROVED** [review]

**Pull request merged**